<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateDialog</class>
 <widget class="QDialog" name="DateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nest</string>
  </property>
  <widget class="QLineEdit" name="dayin">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>  请选择预订日期 : )</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dayout">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>入住日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="text">
    <string>离店日期：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:hover
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(117, 117, 117);
border:2px groove gray;border-radius:10px;padding:2px 4px
}
QPushButton:pressed
{
font: 12pt &quot;华文楷体&quot;;
color:rgb(182, 182, 182)
border:2px groove gray;border-radius:10px;padding:2px 4px
}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
